components/sync15/src/request.rs

```diff
@@ -251,17 +267,21 @@ where
         }
     }
 
+    // UPDATED TO NOT ALLOW UNSUPPORTED BATCHSTATE
+    // Determines if there is a batch
```

Comment:
Returns true iff we are in the middle of uploading a batch.

Reply:
This function is now essentially useless, since there are only two states: a new batch or in the middle of a batch. It can be removed.